specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2017-11-11-preview/examples/SearchIndexPostAutocomplete.json

```diff
@@ -0,0 +1,29 @@
+{
+    "parameters": {
```

Comment:
This looks the same as the GET example. That can't be right -- most of these parameters should be in the request body, no? Did this pass validation?

Reply:
Here's one: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/examples/SearchServiceCreateIndex.json